### PR TITLE
fix: Update org members can edit project ownership rules

### PR DESCRIPTION
**Pull request by jangjodi**
Add that Org Members can edit project ownership rules

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [X] None: Not urgent, can wait up to 1 week+

### CODE REVIEW

File: docs/organization/membership/index.mdx [hunk 1]
```diff
@@ -35,7 +35,7 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can view and act on issues (such as assigning and resolving)                    |             | ✓          | ✓         | ✓           | ✓         |
 | Can join and leave existing teams                                               |             | ✓          | ✓         | ✓           | ✓         |
 | Can add repositories (through GitHub integrations)                              |             | ✓          | ✓         | ✓           | ✓         |
-| Can create and remove teams from the organization                               |             |            | ✓         | ✓           | ✓         |
+| Can create and remove teams from the organization                               |             | ✓\*\*      | ✓         | ✓           | ✓         |
```
Comment (oioki) [inline docs/organization/membership/index.mdx:38]: As this comes before "Can create and remove projects for the organization", should we swap the footnotes? Here will be `✓\*` and `✓\*\*` is below.

Comment (jangjodi) [reply docs/organization/membership/index.mdx]: Good suggestion! I've fixed the ordering

File: docs/organization/membership/index.mdx [hunk 2]
```diff
@@ -49,7 +49,8 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can transfer projects to another organization                                   |             |            |           |             | ✓         |
 | Can remove an organization                                                      |             |            |           |             | ✓         |
 
-\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
+\*Org Members can only edit the project ownership rule setting.
+\*\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
```
Comment (lizokm) [inline docs/organization/membership/index.mdx:53]:
```suggestion
\*Org Members can only edit the project ownership rule setting.

\*\*If your organization is on a Business or Enterprise plan, anyone who's part of the org will be able to create a project without having to be a Team Admin. When org members create a project, they'll become the Team Admin for that project by default and have all the permissions of the [Team Admin](#permissions-1) for that project.
```
@jangjodi I'm not sure I fully understand the difference between being able to add/remove teams versus only being able to edit the rules. Could you clarify? :)

File: docs/organization/membership/index.mdx [hunk 3]
```diff
@@ -113,9 +114,9 @@ Let's say a user is a Team Admin for Team 1 and a Contributor for Team 2, and bo
 
 Projects can only be accessed by that project's team(s). Team Admins, Org Managers, and Org Owners can add or remove project teams in **Project Settings > Project Teams**. Alternatively, these roles can also add projects to a team in **Team Settings > Projects**.
 
-All Org Members can get access to a project by joining a team associated with that project.
+All Org Members can get access to a project by joining a team associated with that project. Additionally, Org Members can edit a project's ownership rules.
```
Comment (lizokm) [inline docs/organization/membership/index.mdx:117]:
```suggestion
Any Org Member can get access to a project by joining the team that's associated with that project. Once you've joined a project you'll also be able to edit [ownership rules](/product/issues/ownership-rules/) for that project.
```

**Review by lizokm — APPROVED** [review]
Thanks for adding. I have one small clarifying question for you :)

File: docs/organization/membership/index.mdx (hunk 2 above)
Comment (jangjodi) [reply docs/organization/membership/index.mdx]: That was actually a typo, thank you for catching @lizokm! I mean to add the check mark in the "Can change project settings" column

File: docs/organization/membership/index.mdx [hunk 4]
```diff
@@ -49,7 +49,9 @@ Organization-level roles enable broad access to the entire organization, as desc
 | Can transfer projects to another organization                                   |             |            |           |             | ✓         |
 | Can remove an organization                                                      |             |            |           |             | ✓         |
 
-\*For Business and Enterprise plans, even Org Members who are not a Team Admin for any team can create a new personal project. When creating this project, the member will get assigned a new personal team that they will become a Team Admin for. [Team Admin permissions](#permissions-1) will then apply for this team.
+\*Org Members can only edit the project ownership rule setting.
```
Comment (lizokm) [inline docs/organization/membership/index.mdx:52]:
```suggestion
\*Org Members will only be able to edit [ownership rules](https://sentry.io/orgredirect/organizations/:orgslug/settings/projects/action-release/ownership/) for the project.
```

**Review by lizokm — APPROVED** [review]

**Pull request merged**